lib/exw3/contracts.ex

```diff
@@ -0,0 +1 @@
+ExW3.DynamoContract.generate_module(ExW3.Contracts.ERC20, "priv/abi/erc20.json")
```

Comment:
I'm just a lib user, don't have any jurisdiction here =P  But I think that maybe you could add a "really dynamic" load here, for the lib and for the parent project itself:
```
"#{File.cwd!}/priv/abi/*.json" # for project path or just "priv/abi/*.json" for lib path or call both
|> Path.wildcard
|> Enum.each(fn path -> 
  name = path |> String.split("/") |> List.last |> String.replace(".json", "") 

  ExW3.Contracts 
  |> Module.concat(String.upcase(name)) # create a dynamic uppercase module name
  |> ExW3.DynamoContract.generate_module(path)  
end)
That would be really useful to have in the project =D

Reply:
Thank you for the suggestion. It depends what we mean by **dynamic**?

I've only added this ERC20 as example here for review I don't know if it would be included in the final PR or not. If we want to do defaults, Your suggestion is very good.